src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -119,6 +119,26 @@
     text: |
         az keyvault create --location westus2 --name MyKeyVault --resource-group MyResourceGroup
     crafted: true
+
+  - name: Create a key vault with network ACLs specified (use --network-acls to specify IP and VNet rules by using a JSON string).
+    text: |
+        az keyvault create --location westus2 --name MyKeyVault --resource-group MyResourceGroup --network-acls "{\\"ip\\": [\\"1.2.3.4\\", \\"2.3.4.0/24\\"], \\"vnet\\": [\\"vnet_name_1/subnet_name1\\", \\"vnet_name_2/subnet_name2\\", \"/subscriptions/000000-0000-0000/resourceGroups/MyResourceGroup/providers/Microsoft.Network/virtualNetworks/MyVNet/subnets/MySubnet\"]}"
+
+  - name: Create a key vault with network ACLs specified (use --network-acls to specify IP and VNet rules by using a JSON file).
+    text: |
+        az keyvault create --location westus2 --name MyKeyVault --resource-group MyResourceGroup --network-acls network-acls-example.json
```

Comment:
No `@` prefix for file?

Reply:
> No `@` prefix for file?

@fengzhou-msft Is `@` a convention? I think in this case users don't need to specify the marker explicitly, the program can detect whether it's a JSON string or a JSON filename.

Reply:
It seems json str or @file.json is a convention, and `type=validate_file_or_dict` can be used, but it makes file completer not working. @Juliehzl @MyronFanQiu any guideline for this?

Reply:
Tested with validate_file_or_dict, it can support json string and file path with or without @ prefix.

Reply:
@fengzhou-msft That's good, I will use `type=validate_file_or_dict` to refine the code.

Reply:
@fengzhou-msft For the help messages and examples, I decided not to add `@` as a prefix since this is not mandatory and may cause confusion.